virtual_ship/instruments/location.py

```diff
@@ -0,0 +1,29 @@
+"""Location class."""
```

Comment:
It is not clear to me yet how you will use this class. Can you explain? or is this for later?

Reply:
The discription for the class is in the docstring of the class. I did not want to repeat the description, so this docstring is fairly useless. It is used in the instruments to define a location on the globe